var/spack/repos/builtin/packages/gapfiller/package.py

```diff
@@ -44,9 +45,20 @@ def url_for_version(self, version):
         return "file://{0}/39GapFiller_v{1}_linux-x86_64.tar.gz".format(
                 os.getcwd(), version.dashed)
 
-    depends_on('perl', type=('build', 'run'))
+    depends_on('perl+threads', type=('build', 'run'))
+
+    def patch(self):
+        with working_dir('.'):
```

Comment:
this shouldn't be necessary, `working_dir('a')` should essentially do `cd a && do stuff && cd olddir`

Reply:
Good point. Thanks.